lib/expression.ex

```diff
@@ -170,7 +170,12 @@ defmodule Expression do
   """
   @spec error(message :: term) :: %{required(String.t()) => term}
   def error(message),
-    do: %{"__type__" => "expression/v1error", "error" => true, "message" => to_string(message)}
+    do: %{
+      "__type__" => "expression/v1error",
```

Comment:
I'm not sure if this was a typo, or if it's the correct value here instead of `"__type__" => "expression/v1"`. @smn Do you know?